<launch>
  <arg name="params_file" default="$(find realtime_line_generator)/config/realtime_viewer.yaml" />

  <node name="realtime_viewer_node" pkg="realtime_line_generator" type="realtime_viewer_node" output="screen">
    <rosparam command="load" file="$(arg params_file)" />
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find realtime_line_generator)/rviz/realtime_viewer.rviz" required="false" />
</launch>
